data/json/items/gun/762.json

```
@@ -115,5 +115,80 @@
     ],
     "magazines": [ [ "762", [ "762x39_clip" ] ] ],
     "flags": [ "RELOAD_ONE", "NEVER_JAMS" ]
+  },
+  {
+    "id": "wasr10",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "WASR-10" },
+    "description": "A semi-automatic-only civilian-legal Kalashnikov-pattern rifle, imported from Romania.  A valuable survivors' commodity.",
+    "weight": "3300 g",
+    "volume": "1750 ml",
```

```suggestion
    "volume": "4487 ml",
```
WASR-10 Image: https://www.centuryarms.com/wasr-144.html
Removal of mag for binarization: https://imgur.com/a/bKt44na
Surface area results(square in, conversion to square cm =631.986 cm^2): https://imgur.com/a/0mgxnGg
Width of 7.1cm using: https://www.gsg-waffen.com/site/index.php/en/ak47-tech-en

 (If @tenmillimaster has a better source on width this can be redone)

---

Is this a new volume deriving method?  That seems awful high, it would put it almost up to the M107's volume.

I think it's in the width you got, they might have included the charging handle in that measurement.  Receiver thickness is ~35mm from what I'm reading, so with the furniture, safety lever, etc., maybe a bit more.  Unfortunately I do not own an AK-pattern rifle that I can put a caliper to.

---

> Is this a new volume deriving method?

It's been used at least since August of last year(#33711), and has been used for all audits done by myself(https://github.com/CleverRaven/Cataclysm-DDA/pulls?q=is%3Apr+author%3ALaVeyanFiend+gun+volume+is%3Aclosed) or tenmillimaster.

>That seems awful high, it would put it almost up to the M107's volume.

The M107 is wrong, as are anything whose volume is 250ml * N.

>Receiver width of 35mm

👀 Source please, and I will redo the calculation for this & the Mini Draco.

---

https://thefirearmsforum.s3.amazonaws.com/2011/09/127656_b8098900fc315bffa17c14246ff5412e.jpg

A work drawing for a homebuild receiver.  Bottom right-most dimension, 1.350" = 3.429cm.  Again, that would be slightly more when off-jutting bits like the safety are considered, even the rivets would add a bit, and I think the handguard is a little wider as well.

https://www.reddit.com/r/ak47/comments/3qzgpc/width_of_akms_underfolding_stock/

This guy claims that the major width of an underfolder stock is about 2.75" or 6.985cm, which significantly juts out from either side of the receiver, for a sanity check.

---

I'll use the work drawing then.
```suggestion
    "volume": "2167 ml",
```